### PR TITLE
fix chinese doc for API

### PR DESCRIPTION
**Pull request by wangchaochaohu**
fix doc
split
image
image
image

index_select
image
image

concat

image
image

### CODE REVIEW

File: doc/fluid/api_cn/tensor_cn/index_select_cn.rst [hunk 1]
```diff
@@ -30,14 +30,14 @@ index_select
...
+        paddle.disable_static()  # Now we are in imperative mode
         data = np.array([[1.0, 2.0, 3.0, 4.0],
                          [5.0, 6.0, 7.0, 8.0],
                          [9.0, 10.0, 11.0, 12.0]])
-        data_index = np.array([-1, 1, 1]).astype('int32')
+        data_index = np.array([0, 1, 1]).astype('int32')
 
-        x = paddle.imperative.to_variable(data)
-        index = paddle.imperative.to_variable(data_index)
+        x = paddle.to_variable(data)
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/index_select_cn.rst]: to_tensor 哈

File: doc/fluid/api_cn/tensor_cn/index_select_cn.rst [hunk 2]
```diff
@@ -30,14 +30,14 @@ index_select
...
+        index = paddle.to_variable(data_index)
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/index_select_cn.rst]: 同理

File: doc/fluid/api_cn/tensor_cn/split_cn.rst [hunk 3]
```diff
@@ -28,7 +28,7 @@ split
     import numpy as np
     import paddle
     
-    paddle.enable_imperative()
+    paddle.disable_static()
     # x is a Tensor which shape is [3, 9, 5]
     x_np = np.random.random([3, 9, 5]).astype("int32")
     x = paddle.imperative.to_variable(x_np)
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/split_cn.rst]: 同理

Comment (wangchaochaohu) [reply doc/fluid/api_cn/tensor_cn/split_cn.rst]: DONE

File: doc/fluid/api_cn/tensor_cn/index_select_cn.rst (hunk 2 above)
Comment (wangchaochaohu) [reply doc/fluid/api_cn/tensor_cn/index_select_cn.rst]: done

File: doc/fluid/api_cn/tensor_cn/index_select_cn.rst (hunk 1 above)
Comment (wangchaochaohu) [reply doc/fluid/api_cn/tensor_cn/index_select_cn.rst]: done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Review by zhangting2020 — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Review by zhangting2020 — APPROVED** [review]
LGTM

**Pull request merged**